_inc/client/components/upgrade-notice-content/index.jsx

```diff
@@ -47,219 +48,70 @@ const UpgradeNoticeContent = moduleSettingsForm(
 			this.props.dismiss();
 		};
 
-		toggleModule = ( name, value ) => {
-			this.props.updateOptions( { [ name ]: ! value } );
-		};
-
-		handleSiteAcceleratorChange = () => {
-			// Initial status for both modules.
-			let newPhotonStatus = this.props.getOptionValue( 'photon' );
-			let newAssetCdnStatus = this.props.getOptionValue( 'photon-cdn' );
-
-			// Check if any of the CDN options are on.
-			const siteAcceleratorStatus = newPhotonStatus || newAssetCdnStatus;
-
-			// Are the modules available?
-			const photonStatus = this.props.getModuleOverride( 'photon' );
-			const assetCdnStatus = this.props.getModuleOverride( 'photon-cdn' );
-
-			// If one of them is on, we turn everything off, including Tiled Galleries that depend on Photon.
-			if ( true === siteAcceleratorStatus ) {
-				if ( false === ! newPhotonStatus && 'active' !== photonStatus ) {
-					newPhotonStatus = false;
-
-					this.props.updateOptions( {
-						photon: false,
-						'tiled-gallery': false,
-						tiled_galleries: false
-					} );
-				}
-				if ( false === ! newAssetCdnStatus && 'active' !== assetCdnStatus ) {
-					newAssetCdnStatus = false;
-
-					this.props.updateOptions( {
-						'photon-cdn': false
-					} );
-				}
-			} else {
-				if ( false === newPhotonStatus && 'inactive' !== photonStatus ) {
-					newPhotonStatus = true;
-
-					this.props.updateOptions( {
-						photon: true,
-						'tiled-gallery': true,
-						tiled_galleries: true
-					} );
-				}
-				if ( false === newAssetCdnStatus && 'inactive' !== assetCdnStatus ) {
-					newAssetCdnStatus = true;
-
-					this.props.updateOptions( {
-						'photon-cdn': true
-					} );
-				}
-			}
-
-			// If at least one of the modules is now on, let's reflect that with the status of our main toggle.
-			if ( true === newPhotonStatus || true === newAssetCdnStatus ) {
-				// Track the main toggle switch.
-				analytics.tracks.recordJetpackClick( {
-					target: 'jetpack_site_accelerator_toggle',
-					toggled: 'on',
-					from: 'warm_welcome'
-				} );
-			} else {
-				analytics.tracks.recordJetpackClick( {
-					target: 'jetpack_site_accelerator_toggle',
-					toggled: 'off',
-					from: 'warm_welcome'
-				} );
-			}
-
-			// Track any potential Photon toggle switch.
-			if ( this.props.getOptionValue( 'photon' ) !== newPhotonStatus ) {
-				analytics.tracks.recordEvent( 'jetpack_wpa_module_toggle', {
-					module: 'photon',
-					toggled: ( false === newPhotonStatus ) ? 'off' : 'on',
-					from: 'warm_welcome'
-				} );
-			}
-
-			// Track any potential Photon CDN toggle switch.
-			if ( this.props.getOptionValue( 'photon-cdn' ) !== newAssetCdnStatus ) {
-				analytics.tracks.recordEvent( 'jetpack_wpa_module_toggle', {
-					module: 'photon-cdn',
-					toggled: ( false === newAssetCdnStatus ) ? 'off' : 'on',
-					from: 'warm_welcome'
-				} );
+		renderLearnMore = () => {
+			const versionSupportsGutenberg = /(5\.0).*/;
```

Comment:
Rather than checking for version, what if we instead passed `Jetpack_Gutenberg::is_gutenberg_available()` over with the initial state? The reason I suggest this is to catch the cases where someone might be running 4.9 with the Gutenberg plugin, as I was when I tested it. Not a blocker to merge.

Reply:
I created #10696 to address this.